### PR TITLE
solves issue #3

### PR DESCRIPTION
**Pull request by arademaker**
this solves issue #3

### CODE REVIEW

**Comment by arademaker** [comment]
@dan-zeman sorry for the mistakes with the branches. Hoppe that now everything is right. The issue-3 branch is based on `dev` and this PR is to merge into `dev`.

**Comment by arademaker** [comment]
one extra commit to solve #2 too.

**Comment by arademaker** [comment]
@dan-zeman I will wait you merge this PR, or confirm that I can make it myself, before solving the remaining issue #1

File: pt_pud-ud-test.conllu [hunk 1]
```diff
@@ -157,15 +157,15 @@
 13	mas	_	CCONJ	CC	_	19	cc	_	_
 14	o	_	DET	DT	Gender=Masc|Number=Sing	15	det	_	_
 15	destino	_	NOUN	NN	Gender=Masc|Number=Sing	19	nsubj	_	_
-16-17	dos	_	_	_	_	_	_	_	_
+16-17	da	_	_	_	_	_	_	_	_
 16	de	de	ADP	INDT	_	18	case	_	_
-17	os	o	DET	_	Gender=Masc|Number=Plur	18	det	_	_
-18	republicanos	_	NOUN	NN	Gender=Masc|Number=Plur	15	nmod	_	_
+17	a	o	DET	_	Gender=Fem|Number=Plur	18	det	_	_
```
Comment (dan-zeman) [inline pt_pud-ud-test.conllu:162]: Shouldn't _a_ be `Number=Sing`?

**Review by dan-zeman — COMMENTED** [review]
In general looks good to me but see my comment at baef74f56cba83033d0fc4117fc3181c84c464a3

File: pt_pud-ud-test.conllu (hunk 1 above)
Comment (arademaker) [reply pt_pud-ud-test.conllu:162]: solved. it is always good to have a second person double-checking.

**Comment by arademaker** [comment]
Can I accept it now?

**Comment by dan-zeman** [comment]
Looks good to me now :-)

**Comment by dan-zeman** [comment]
... but it looks like I overlooked a text mismatch error :-(
Too bad that we cannot run the validator on pull requests before merging them.

**Comment by dan-zeman** [comment]
Fixed in 5191c52173d00b9d6cebf211dd66ab3e72247ad9

**Pull request merged**